src/JHipsterNetSampleApplication/Controllers/BankAccountController.cs

```diff
@@ -73,9 +75,9 @@ public ActionResult<IEnumerable<BankAccount>> GetAllBankAccounts(IPageable pagea
         public async Task<IActionResult> GetBankAccount([FromRoute] long id)
         {
             _log.LogDebug($"REST request to get BankAccount : {id}");
-            var result =
-                await _applicationDatabaseContext.BankAccounts.SingleOrDefaultAsync(bankAccount =>
-                    bankAccount.Id == id);
+            var result = await _applicationDatabaseContext.BankAccounts
+                .Include(bankAccount => bankAccount.User)
```

Comment:
Just asking (I didn't test it) if we can 
1- to the filter condition (SingleOrDefaultAsync)
2- the projection (Include)
It makes more sense to me to do it in this order.

Reply:
Yes, I agree with you, I tested it and it doesn't work.
It is always in this order with LINQ queries.